spec/services/available_accounts_finder_spec.rb

```diff
@@ -30,6 +30,19 @@
         is_expected.to eq([chartstring])
       end
     end
+
+    context "with a different current user" do
+      subject(:accounts) { described_class.new(user, facility, current: current, current_user: current_user).accounts }
+
+      let(:current_user) { FactoryGirl.create(:user) }
+      let!(:other_chartstring) { FactoryGirl.create(:nufs_account, :with_account_owner, owner: current_user) }
+
+      before { FactoryGirl.create(:account_user, :business_administrator, user: user, account: other_chartstring) }
+
+      describe "for any facility" do
+        it { is_expected.to match_array([chartstring, other_chartstring]) }
```

Comment:
A minor style thing to consider (which I often forget): this could be written as `is_expected.to contain_exactly(chartstring, other_chartstring)` to avoid the `[]`.